Assignment3/tinotrifan-y160077/Assigment3.txt

```diff
@@ -0,0 +1,276 @@
+http://www.w3.org/2000/01/rdf-schema#label	"أكسل أكسنسترنا"@ar
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@de
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@es
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@fr
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@it
+http://www.w3.org/2000/01/rdf-schema#label	"アクセル・オクセンシェルナ"@ja
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@nl
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@pl
+http://www.w3.org/2000/01/rdf-schema#label	"Axel Oxenstierna"@pt
+http://www.w3.org/2000/01/rdf-schema#label	"Оксеншерна, Аксель"@ru
+http://www.w3.org/2000/01/rdf-schema#label	"Brothers Grimm"@en
+http://www.w3.org/2000/01/rdf-schema#label	"الأخوان غريم"@ar
+http://www.w3.org/2000/01/rdf-schema#label	"Brüder Grimm"@de
+http://www.w3.org/2000/01/rdf-schema#label	"Hermanos Grimm"@es
+http://www.w3.org/2000/01/rdf-schema#label	"Frères Grimm"@fr
+http://www.w3.org/2000/01/rdf-schema#label	"Fratelli Grimm"@it
+http://www.w3.org/2000/01/rdf-schema#label	"グリム兄弟"@ja
+http://www.w3.org/2000/01/rdf-schema#label	"Gebroeders Grimm"@nl
+http://www.w3.org/2000/01/rdf-schema#label	"Bracia Grimm"@pl
+http://www.w3.org/2000/01/rdf-schema#label	"Irmãos Grimm"@pt
+http://www.w3.org/2000/01/rdf-schema#label	"Братья Гримм"@ru
+http://www.w3.org/2000/01/rdf-schema#label	"格林兄弟"@zh
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Menem"@en
+http://www.w3.org/2000/01/rdf-schema#label	"كارلوس منعم"@ar
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Menem"@de
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Menem"@es
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Menem"@fr
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Saúl Menem"@it
+http://www.w3.org/2000/01/rdf-schema#label	"カルロス・メネム"@ja
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Menem"@nl
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Saúl Menem"@pl
+http://www.w3.org/2000/01/rdf-schema#label	"Carlos Menem"@pt
+http://www.w3.org/2000/01/rdf-schema#label	"Менем, Карлос"@ru
+http://www.w3.org/2000/01/rdf-schema#label	"卡洛斯·梅内姆"@zh
+http://www.w3.org/2000/01/rdf-schema#label	"Carter Harrison Jr."@en
+http://www.w3.org/2000/01/rdf-schema#label	"كارتر هاريسون"@ar
+http://www.w3.org/2000/01/rdf-schema#label	"Carter Harrison, Jr."@de
+http://www.w3.org/2000/01/rdf-schema#label	"Carter Harrison Sr."@en
+http://www.w3.org/2000/01/rdf-schema#label	"Carter Harrison, Sr."@de
+
+
+#########################################################################################################################
+#########################################################################################################################
+
+
+5. For each of the properties, except rdf:type, that can be applied to instances of the Politician class, how many
+distinct values do they take in those instances?
+
+SELECT DISTINCT ?property COUNT(?property) AS ?distinctValues
```

Comment:
This is not right, since you are counting on the property again, not on the values.

Reply:
fixed